Sources/StreamFeeds/Models/BookmarkFolderData.swift

```diff
@@ -26,3 +28,17 @@ extension BookmarkFolderResponse {
         )
     }
 }
+
+// MARK: - Local Filter Matching
+
+extension BookmarkFolderData {
+    struct LocalFilterData: Equatable, Sendable {
+        var userId: String = ""
```

Comment:
In Android I'm removing userId as filter field because, for client requests, the [backend infers it from the token](https://github.com/GetStream/chat/blob/52bbf085412485ad55e8bf5e033384c0efa3d0dc/controllers/feeds/query_bookmark_folders.go#L42)